### DISCUSSION TITLE
Compiling With An pacakge.json Using Netlify Makes My Game Stuck At 0% (https://618c72251abce500079dea70--elegant-swartz-049c3c.netlify.app/)

### DISCUSSION THREAD

**Discussion by DaringAce** [FunkinCrew/Funkin#1877] [Compiling help]
I'm not a code expert or anything, but how come compiling with an package.json using netlify makes my game stuck at 0% (https://618c72251abce500079dea70--elegant-swartz-049c3c.netlify.app/)?

**Comment by AbnormalPoof** [comment]
you stole the game ew ew ew ew ew ew



missing assets

**Reply by DaringAce** [reply]
bruh the game is open source for a reason

**Comment by libauror-a** [comment]
i just looked into the sources and...

![image](https://user-images.githubusercontent.com/75984453/142277375-902b45d3-cbfe-4f3e-9446-e472fa6c9b60.png)

fonts

**Reply by RevixDev** [reply]
xd
